sbin/build.sh

```diff
@@ -478,9 +478,9 @@ makeACopyOfLibFreeFontForMacOSX() {
 getFirstTagFromOpenJDKGitRepo()
 {
     git fetch --tags "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}"
-    justOneFromTheRevList=$(git rev-list --tags --max-count=1)
-    tagNameFromRepo=$(git describe --tags "$justOneFromTheRevList")
-    echo "$tagNameFromRepo"
+    RevList=$(git rev-list --tags --max-count=100)
+    firstMatchingNameFromRepo=$(git describe --tags "$RevList" | grep ^jdk | head -1)
```

Comment:
Hmmmm, fairly big assumption to make.  I'd want to see this get extracted to a labeled const RELEASE_TAG_PREFIX or similar.

We do also have the following release tag in aarch64 `aarch64-jdk8u181-b13` - although we deliberately did rewrite that as `jdk8u181-b13` - see https://github.com/AdoptOpenJDK/openjdk-aarch64-jdk8u/releases

Reply:
Have relaxed it to ensure it has "jdk" anywhere but excludes `openj9` ... I could have just excluded `openj9` but for now this felt better in case we get and `jdk8u192-openj9` or similar tags that we want to avoid